Refactor layout to align things properly, add a close button.

miguelandres opened the pull request.

Commit:
Refactor layout to align things properly, add a close button.

pingpongboss commented on remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/view/RemixerFragment.java:
```diff
@@ -146,7 +143,16 @@ public View onCreateView(LayoutInflater inflater,
+    closeButton.setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View view) {
+        FragmentManager manager = getFragmentManager();
```
nit: inline

pingpongboss commented on remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/view/RemixerFragment.java:
```diff
@@ -146,7 +143,16 @@ public View onCreateView(LayoutInflater inflater,
+        FragmentManager manager = getFragmentManager();
+        FragmentTransaction transaction = manager.beginTransaction();
+        transaction.remove(RemixerFragment.this);
+        transaction.commit();
```
nit: chain

miguelandres replied:
Done

pingpongboss commented on remixer_ui/src/main/java/com/google/android/libraries/remixer/ui/widget/BooleanVariableWidget.java:
```diff
@@ -32,7 +33,7 @@
  */
 @Keep
-public class BooleanVariableWidget extends RelativeLayout
+public class BooleanVariableWidget extends LinearLayout
```
suggest you explicitly set the orientation in xml/java. Suggest java.

miguelandres replied:
I think it's harder to read in Java, but agreed. I'll put it on xml though.

Commit:
Fix tests and reply to Mark’s comments:

1. add a project.properties file to remixer_ui/src/main/ so Robolectric knows to look at Support Library’s code for styles.
2. Set a theme across AndroidManifest files and the corresponding tools:replace in the app’s manifest.
3. Inline close code.
4. explicitly state the horizontal orientation on BooleanVariableWidget.
5. Add remixer text styles.
6. Format xml files (sort parameters and so forth).
